### DISCUSSION TITLE
Immich instalaltion and remote storage

### DISCUSSION THREAD

**Discussion by Psykozo51** [immich-app/immich#9927] [Q&A] [answered]
Hi
After a docker deploiement, i will add 3 users (my familly) and save our photos etc on ich folder on my NAS shared
In my compose I add my mount point on my nas with user authentification
It's possible to add a remote storage on the webui ton configure the backup for ich user ?
Thansk for your respons

**Comment by Meliox** [comment]
Read here. 
https://immich.app/docs/guides/external-library/

Docker compose example
services:
  immich-server:
    container_name: immich_server
    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
    command: [ "start.sh", "immich" ]
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      _- localLibrary:/mnt/Photos/_
      - /root/shared/:/shared      
.....

**volumes:

  localLibrary:
    driver_opts:
      type: cifs
      device: //192.168.3.4/Photos
      o: addr=192.168.3.4,username=ghjbvg,password=kkhh,file_mode=0555,dir_mode=0555,vers=3.0**

Mounted as read  only. I

**Reply by Psykozo51** [reply]
ok, but after, I add this folder on the webui under external library, but after, how can I backup my photo under this folder?

**Reply by Meliox** [reply]
I have actually not read about anyone using  a) the normal upload function on external storage and b) in particular not the same.

It should be possible to do the same with a... 

I'll leave that use case for others to answer.

**Reply by Psykozo51** [reply, marked as answer]
hi, ok after a lot of search and many tests, I change the UPLOAD_LOCATION and I mount a shared folder in volume configuration on the compose file

**Comment by Psykozo51** [comment]
To sum up, I would like to add a volume to mount my share on my NAS.
Add a user for my wife, for example, and when she installs the app on her mobile, the photos she saves will be saved directly to the share.
